param/parameterized.py

```diff
@@ -2698,6 +2713,32 @@ def pprint(self, imports=None, prefix="\n    ",unknown_value='<?>',
 
 
 
+class default_label_formatter(ParameterizedFunction):
+    "Default formatter to turn parameter names into appropriate widget labels."
+
+    capitalize = Parameter(default=True, doc="""
+        Whether or not the label should be capitalized.""")
+
+    replace_underscores = Parameter(default=True, doc="""
+        Whether or not underscores should be replaced with spaces.""")
+
+    overrides = Parameter(default={}, doc="""
+        Allows custom labels to be specified for specific parameter
+        names using a dictionary where key is the parameter name and the
+        value is the desired label.""")
+
+    def __call__(self, pname):
+        if pname in self.overrides:
+            return self.overrides[pname]
+        if self.replace_underscores:
+            pname = pname.replace('_',' ')
+        if self.capitalize:
+            pname = pname[:1].upper() + pname[1:]
+        return pname
+
+
+label_formatter = default_label_formatter.instance()
```

Comment:
What is the benefit of `.instance()` here?

Reply:
Just that I would expect `.instance` to be used to set the `label_formatter` and did the same internally. e.g:

```
param.parameterized.label_formatter = default_label_formatter.instance(capitalized=True)
```